src/components/HearingList.js

```diff
@@ -9,6 +14,47 @@ import LabelList from './LabelList';
 import Label from 'react-bootstrap/lib/Label';
 import LoadSpinner from './LoadSpinner';
 import getAttr from '../utils/getAttr';
+import HearingsSearch from './HearingsSearch';
+import toLower from 'lodash/toLower';
+import upperFirst from 'lodash/upperFirst';
+
+
+const HearingListTabs = () =>
+  <Nav className="hearing-list__tabs" bsStyle="tabs" style={{width: '100%'}} activeKey="1" pullRight={1}>
+    <NavItem eventKey="3" disabled style={{width: '70%'}}/>
+    <NavItem eventKey="1" href="/home">Lista</NavItem>
+    <NavItem eventKey="2" title="Item">Kartta</NavItem>
+  </Nav>;
+
+const HearingListFilters = ({labels, handleChangeFilter, handleSort}) =>
+  <div className="hearing-list__filter-bar">
+    <FormGroup controlId="formControlsSelect" className="hearing-list__filter-bar-filter">
+      <ControlLabel><FormattedMessage id="sort"/></ControlLabel>
+      <FormControl componentClass="select" placeholder="select" onChange={(event) => handleSort(event.target.value)}>
+        <option value="-created_at"><FormattedMessage id="newestFirst"/></option>
+        <option value="created_at"><FormattedMessage id="oldestFirst"/></option>
+        <option value="-close_at"><FormattedMessage id="lastClosing"/></option>
+        <option value="close_at"><FormattedMessage id="firstClosing"/></option>
+        <option value="-open_at"><FormattedMessage id="lastOpen"/></option>
+        <option value="open_at"><FormattedMessage id="firstOpen"/></option>
+        <option value="-n_comments"><FormattedMessage id="mostCommented"/></option>
+        <option value="n_comments"><FormattedMessage id="leastCommented"/></option>
+      </FormControl>
+    </FormGroup>
+    {labels.length !== 0 && <FormGroup controlId="formControlsSelect" className="hearing-list__filter-bar-filter">
+      <ControlLabel><FormattedMessage id="chooseTopic"/></ControlLabel>
+      <FormControl componentClass="select" placeholder="select" onChange={(event) => handleChangeFilter(event.target.value)}>
```

Comment:
I have the feeling drop-down menu doesn't really cut it in here. There are too many possible labels to select from, and there will be even more in the future.

Currently there is no autocomplete API endpoint for fetching labels when typing (though there could be), but how about a text box (below the title search text box) that allows you to start typing and select matching label from the autocomplete menu? This would allow you to select several labels.